lib/canvas_yaml.rb

```diff
@@ -28,6 +28,7 @@
 if defined?(YAML::ENGINE)
```

Comment:
Ah, yes, and in looking into that, I found that there is one other thing that needs to be added here.
`Object#to_yaml` also needs to be re-patched. I'll drop the dead code and add the to_yaml fix.